docs/types.rst

```diff
@@ -986,9 +988,7 @@ is possible if it
 makes sense semantically and no information is lost: ``uint8`` is convertible to
 ``uint16`` and ``int128`` to ``int256``, but ``int8`` is not convertible to ``uint256``
 (because ``uint256`` cannot hold e.g. ``-1``).
-Furthermore, unsigned integers can be converted to bytes of the same or larger
-size, but not vice-versa. Any type that can be converted to ``uint160`` can also
-be converted to ``address``.
+Any type that can be converted to ``uint160`` can also be converted to ``address``.
```

Comment:
Is that still true?

Reply:
For integer types, yes (which is what I verified), but not for literals, this should be more explicit here. And address types should probably be mentioned in the section below about literal conversions as well.

Or maybe we should change this for integer types as well?

Currently the following works:
```
contract C {
  function f() public pure returns (address) {
    return uint8(2);
  }
}
```

Reply:
Yeah, we should decouple address from integer, but not sure if this will be part of 0.5.0. If we add `address payable`, that is probably the right time.